src/chocolatey.resources/helpers/functions/Install-ChocolateyShortcut.ps1

```diff
@@ -1,4 +1,4 @@
-# Copyright © 2017 Chocolatey Software, Inc.
+﻿# Copyright © 2017 Chocolatey Software, Inc.

```

Comment:
UTF-8 change here? 

Earlier I mentioned it was fine most of the time - this is one case where we must absolutely have UTF-8 and CRLF or Authenticode signing is invalidated.

Reply:
I verified that everything is in order here.  The file is UTF-8-BOM, same as all the other files in the folder, and all line feeds match,